opentelemetry-otlp/Cargo.toml

```diff
@@ -67,12 +67,14 @@ grpc-tonic = ["tonic", "prost", "http", "tokio", "opentelemetry-proto/gen-tonic"
 gzip-tonic = ["tonic/gzip"]
 tls = ["tonic/tls"]
 tls-roots = ["tls", "tonic/tls-roots"]
+tls-webkpi-roots = ["tls", "tonic/tls-webpki-roots"]
 
 # http binary
 http-proto = ["prost", "opentelemetry-http", "opentelemetry-proto/gen-tonic-messages", "http", "trace", "metrics"]
 reqwest-blocking-client = ["reqwest/blocking", "opentelemetry-http/reqwest"]
 reqwest-client = ["reqwest", "opentelemetry-http/reqwest"]
 reqwest-rustls = ["reqwest", "reqwest/rustls-tls-native-roots"]
+reqwest-rustls-webkpi-roots = ["reqwest", "reqwest/rustls-tls-webpki-roots"]
```

Comment:
I think it would be better to also add these two features - `reqwest-rustls` and `reqwest-rustls-webkpi-roots` inside `opentelemetry-http`, and then use it here:
```rust
reqwest-rustls = ["opentelemetry-http/reqwest-rustls"]
reqwest-rustls-webkpi-roots = ["opentelemetry-http/reqwest-rustls-webkpi-roots"]
```

Reply:
that makes sense, thanks for the context! added the changes in the next commit